doc/Programming-guide/Key-platform-features/Data-collection/index.md

```diff
@@ -10,55 +10,51 @@ sort_idx: 50
 * TOC
 {:toc}
 
-The Kaa data collection subsystem is designed to collect records (logs) of pre-configured structure on the endpoints, periodically transferring these records
-from endpoints to Operations service, and, finally, either persisting them on the server for further processing or submitting them to immediate stream
-analysis. The log structure in Kaa is determined for each application by a configurable log schema. On the Operations service side, there are log appenders
-which are responsible for writing logs received by the Operations service into the specific storage. It is possible to add several log appenders to work
-simultaneously.
+The data collection subsystem in Kaa is designed to collect records (logs) of pre-configured structure, store them in the [endpoint]({{root_url}}Glossary/#endpoint-ep), transfer them from the endpoint to [Operations service]({{root_url}}Glossary/#operations-service), persist them on server for further processing, and submit them to the immediate stream analysis.
```

Comment:
If you say that logs stored in the endpoint, please add link to the type of storage. "... , [store](#log-storage) them in the ..." because it looks like logs stored in the EP as they are stored in the specific storage place depending on the log appender type.